storage/src/main/java/tech/pegasys/teku/storage/server/VersionedDatabaseFactory.java

```diff
@@ -40,25 +40,33 @@
   private final File dbVersionFile;
   private final StateStorageMode stateStorageMode;
   private final DatabaseVersion createDatabaseVersion;
+  private final long stateStorageFrequency;
 
   public VersionedDatabaseFactory(
       final MetricsSystem metricsSystem,
       final String dataPath,
       final StateStorageMode dataStorageMode) {
-    this(metricsSystem, dataPath, dataStorageMode, DatabaseVersion.DEFAULT_VERSION.getValue());
+    this(
+        metricsSystem,
+        dataPath,
+        dataStorageMode,
+        DatabaseVersion.DEFAULT_VERSION.getValue(),
+        2048L);
```

Comment:
nit: It might be a good idea to make this a class level constant.

Reply:
most of the time when i do that it gets removed anyway, but I'm happy to :/